src/features/Home/TipsModal/index.tsx

```
@@ -77,69 +77,92 @@ const TipsModal = ({
+                            borderRadius: '12px 12px 0 0',
+                            borderTop: `1px solid ${theme.gray[100]}`,
+                          }),
+                          ...(Tips?.mails.indexOf(item) ===
+                            Tips?.mails.length - 1 && {
+                            borderRadius: '0 0 12px 12px',
+                            borderBottom: `1px solid ${theme.gray[100]}`,
+                          }),
+                        }}
+                      >
+                        <div className={s.Email_From}>{item.from}</div>
+                        <div className={s.Email_Header}>{item.subject}</div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)